support/environment_test.go

```diff
@@ -0,0 +1,82 @@
+package support
+
+import (
+	"os"
+	"testing"
+)
+
+func TestGetCodeFlareSDKVersion(t *testing.T) {
```

Comment:
Honestly I think that the simple getters in `environment.go` don't have much sense to be tested.
Possibly it may have sense to test `GetClusterType` - https://github.com/project-codeflare/codeflare-common/blob/main/support/environment.go#L84

Reply:
I understand , will include test for `GetClusterType` . Thanks for the suggestion

Reply:
Added unit test for `GetClusterType` . Feel free to reveiew it